sunpy/instr/goes.py

```diff
@@ -41,7 +41,7 @@
 ----------
 
 .. [Ref1] Hanser, F.A., & Sellers, F.B. 1996, Proc. SPIE, 2812, 344
-.. [Ref2] Dere, K.P., et al. 2009 A&A, 498, 915 DOI: `10.1051/0004-6361/200911712 <https://dx.doi.org/10.1051/0004-6361/200911712>`__
+.. [Ref2] Dere, K.P., et al. 2009 A&A, 498, 915 DOI: `10.1051/0004-6361/200911712 <https://www.aanda.org/articles/aa/abs/2009/18/aa11712-09/aa11712-09.html>`__
```

Comment:
Are we going to replace all dx.doi.org urls that mean to be permanent to their journal urls?? I'm not sure this is a good idea...

Reply:
Ok! I didn't realize that dx.doi was used for that. I just was changing all redirections I found in the log.
But now, something come up to me: if we don't change every link reported by the log, we'll always have warnings in it, and it will be more difficult to clean the list of real problems. I don't know how to deal with it. We need a kind of whitelist

Reply:
Yes, we need to whitelist these... I don't know how. maybe @Cadair who look into that knows...